README.md

```diff
@@ -24,14 +24,26 @@ The `deepstack-ui` is designed to be run in a docker container. The UI picks up
 - DEEPSTACK_UI_DEBUG_MODE : options `True` or `False` (default). Lowers the minimum confidence threshold to 1%
 ```
 
-From the root dir, build the deepstack-ui container from source and then run the UI, passing the `DEEPSTACK_IP` environment variable:
+From the root dir, either build the deepstack-ui container from source or pull from Docker Hub: and 
+
+For x86-64 architecture
 ```
-    docker build -t deepstack-ui .
+    docker build -t deepstack-ui . -f Dockerfile
     OR
     docker pull robmarkcole/deepstack-ui:latest
+```
+
+Or for arm64 architectue (Raspberry Pi)
```

Comment:
Maybe the filename should be `Dockerfile.raspberry` Apple M1 also is arm

Reply:
also typo in architectue

Reply:
M1 is aarch64 / arm64, so it should run on it too, right? Haven't played with those new fancy toys yet

Reply:
Created issue for M1